firestore/cloud-client/snippets.py

```diff
@@ -990,3 +1016,42 @@ def regional_endpoint():
         print(r)
     # [END firestore_regional_endpoint]
     return cities_query
+
+
+def query_filter_compound_multi_ineq():
+    from google.cloud import firestore
+
+    db = firestore.Client()
```

Comment:
Can we make the import and client creation as part of the sample to make it copy-paste-runnable?

Same with other samples.